### DISCUSSION TITLE
integration

### DISCUSSION THREAD

**Discussion by Jesse-Lucas1996** [NetrisTV/ws-scrcpy#157] [Q&A]
Hello! 

I am developing a mass scale service that utilizes ADB to connect to over 1000 TVS, I would love to use this as well!
How could I possibly integrate this into an existing service!
thank you so much!

**Comment by drauggres** [comment]
Hi.
To be honest, I never planned that this project would be used as part of another. You can try to add it as git submodule and reuse code directly from this repository or just copy necessary parts.

Let me know if you have any ideas what could be improved for easier integration.

**Reply by Jesse-Lucas1996** [reply]
Awesome, thank you so much! I will keep this updated if you wish with the progress?

**Reply by drauggres** [reply]
Yeah, sure. Keep me posted :-).
